File: models/tipoLixoAceitoEcoponto.js
module.exports = (sequelize, DataTypes) => {
  const TipoLixoAceitoEcoponto = sequelize.define('TipoLixoAceitoEcoponto', {
    pontos_kg: DataTypes.DOUBLE,
    img: DataTypes.STRING,
  }, {
    tableName: 'tb_tipo_lixo_aceito_ecoponto',
    timestamps: false,
  });

  TipoLixoAceitoEcoponto.associate = (models) => {
    TipoLixoAceitoEcoponto.belongsTo(models.TipoLixo, { foreignKey: 'id_tipo_lixo' });
    TipoLixoAceitoEcoponto.belongsTo(models.Ecoponto, { foreignKey: 'id_ecoponto' });
  };

  return TipoLixoAceitoEcoponto;
};

File: src/main/java/org/acme/models/TipoLixoAceitoEcoponto.java
package org.acme.models;
import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "tb_tipo_lixo_aceito_ecoponto")
public class TipoLixoAceitoEcoponto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    @Column(name = "pontos_kg")
    private Double pontosKg;

    @Column(name = "img")
    private String img;

    @Column(name = "aceito")
    private Boolean aceito;

    @ManyToOne
    @JoinColumn(name = "id_ecoponto")
    private Ecoponto ecoponto;

    @ManyToOne
    @JoinColumn(name = "id_tipo_lixo")
    private TipoLixo tipoLixo;
}